addon/session-stores/adaptive.js

```diff
@@ -66,7 +69,7 @@ export default Base.extend({
     @type Integer
     @public
   */
-  cookieExpirationTime: null,
+  cookieExpirationTime: computed.alias('_store.cookieExpirationTime'),
```

Comment:
I think these properties have to be actual properties on the adaptive store itself as they are read from that and assigned to the cooke store when that's created (see [line 92](https://github.com/simplabs/ember-simple-auth/pull/983/files#diff-8ccd5433965bb44e3dbb5f054de89699R92))